app/src/main/res/layout/activity_auth.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    tools:context=".AuthActivity">
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:id="@+id/topPanel"
+        android:layout_height="wrap_content"
+        android:layout_weight="1">
+
+        <TextView
+            android:id="@+id/anchorTop"
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            android:layout_centerInParent="true"/>
+
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="LOGIN"
+            android:textColor="#3f3f44"
+            android:textSize="32sp"
```

Comment:
I suppose this should be using values from 48828670eee5126f6ac05a92e4c9823496e2ce4a?

Reply:
Ugh I think I should convert this PR to draft so that you won't merge it again.

Reply:
LOL no use, I can convert it back :smiling_imp:  I don't have the habit of merging until everything is figured out though; a blank patch on the other hand is very straightforward.